utils.go

```diff
@@ -2,31 +2,40 @@ package pgcommands
 
 import (
 	"bufio"
+	"errors"
 	"fmt"
 	"io"
 	"os/exec"
 )
 
 type ExecOptions struct {
-	StreamPrint bool
+	StreamPrint       bool
+	StreamDestination io.Writer
```

Comment:
Maybe we can make the structure lighter with a pointer and not print when we have a nil value.

Reply:
We could remove the `StreamPrint` property from the `ExecOptions` struct and change the stream condition in the `utils.go` file [here](https://github.com/dhis2-sre/pg-commands/blob/dev/utils.go#L28) to `if options.StreamDestination != nil {`.

But I'm wondering if that's really a good idea and if it isn't better to be explicit about what we want to do. I'm honestly not sure what's the best way. Go seems to prefer being explicit about intentions and not having any "magic".

Reply:
Ok that's right, so let's keep "StreamPrint" params